### PR TITLE
IDE-2885 improve gradle checking

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

**Comment by yanan** [comment]
Test Failed on Jenkins http://ide-build-1:9090/ on this commit d74d0c70c7c71b6cf92e7fc04b114c3bed651801 on build 89

File: tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java [hunk 1]
```diff
@@ -154,9 +159,48 @@ private static void configureIfLiferayProject( final IProject project, final Gra
...
             Job job = new WorkspaceJob( "Checking gradle configuration" )
             {
+
                 @Override
                 public IStatus runInWorkspace( IProgressMonitor monitor ) throws CoreException
                 {
+                    IFile bndFile = project.getFile( "bnd.bnd" );
+
+                    //case 1: has bnd file
+                    if( bndFile != null && bndFile.exists() )
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java]: We need to check for buildship nature as well (that means its a gradle project)

Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java]: yeah , we already have the detection , see 
https://github.com/liferay/liferay-ide/blob/master/tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java#L153

Comment (gamerson) [inline tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java]: Oh I see, but lets add those additional checks outside of the Job.  So just move the checks for bnd.bnd and gulpfile outside of the job.  But keep the nature modifications and gradle tooling model checks in the job.

thanks.

Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java]: okay , got it , I will send another pr to fix this.

Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java]: sorry . Greg,  it is hard to extract the checks out of a job . cause  I use a var needAddNaure outside job , and I need to change the value of the var , but when I want to read the var in job, it need the var to be final. So I think it doesn't matter if checks inside or outside the job.

**Pull request closed without merging**